File: Lab4/main/java/window/view/WindowView.java
package window.view;

public class WindowView {
    public void printStartMenu() {
        System.out.println("\nEnter a number to do a command:");
        System.out.println("1. Create sentence by yourself");
        System.out.println("2. Create sentence by Lorem");
        System.out.println("3. Open window and show its properties");
        System.out.println("4. Proxy example");
        System.out.println("0. Close window");
        System.out.print("\nInput a number: ");
    }

    public void printErrorMessage() {
        System.out.println("Type correct number!");
    }

    public void printSelfCreation() {
        System.out.println("Enter you sentence");
    }

    public void printLoremCreation() {
        System.out.print("Input a number of words: ");
    }

}

File: Lab6/src/main/kotlin/view/WindowView.kt
package view

class WindowView {
    fun printStartMenu() {
        println("\nEnter a number to do a command:")
        println("1. Create sentence by yourself")
        println("2. Create sentence by Lorem")
        println("3. Open window and show its properties")
        println("0. Close window")
        print("\nInput a number: ")
    }

    fun printErrorMessage() {
        println("Type correct number!")
    }

    fun printSelfCreation() {
        println("Enter you sentence")
    }

    fun printLoremCreation() {
        print("Input a number of words: ")
    }
}